command/crypto/jws/sign.go

```diff
@@ -30,6 +32,8 @@ be written to STDOUT.
 
 For examples, see **step help crypto jws**.`,
 		Flags: []cli.Flag{
+			flags.X5cCert,
+			flags.X5cKey,
```

Comment:
Order is important, move them down.

Reply:
Reuse `--key`? Whatever looks better.

Reply:
I can make it "key, x5c-key", but I would like to have `x5c-key` available since we're using that in `step ca certificate`.